<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerupdater</class>
 <widget class="QDialog" name="customerupdater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color: rgb(34, 40, 49);
	
}



QDialog
{
	background-color: rgb(238, 238, 238);
}



QLabel
{
	background-color:rgb(238, 238, 238);
	color: rgb(34, 40, 49);
	font: 11pt &quot;DejaVu Sans Mono&quot;;
}

QLineEdit
{

	color:rgb(238, 238, 238);
	background-color: rgb(34, 40, 49);
	border-radius: 9px;
	font: 9pt &quot;DejaVu Sans Mono&quot;;

}


QPushButton
{
	border-radius: 12px;
	background-color: rgb(255, 211, 105);
	color: rgb(34, 40, 49);
	font: 700 11pt &quot;DejaVu Sans Mono&quot;;
	
	
}




</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>0</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Ubuntu&quot;;
background-color: rgb(34, 40, 49);
color:rgb(238, 238, 238);
</string>
    </property>
    <property name="text">
     <string>Customer Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorCurrentNumber">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="creditIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="updateCustomer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>640</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCustomer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearScreen">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>690</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentnumberIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Number</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewAddress">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewCredit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>16</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorNewNumber">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line2_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>260</y>
     <width>1300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70) ;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="updatedSucessfully">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>740</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>creditIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>addressIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>nameIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>numberIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>errorCurrentNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorCurrentNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>currentnumberIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentnumberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorCurrentNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorNewNumber</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentnumberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>updatedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>752</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>updatedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>708</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>749</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
